float32 bola_x
float32 bola_y
float32 motor1
float32 motor2
float32 motor3